integration-libs/segment-refs/root/feature-name.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const SEGMENT_FEATURE = 'Segment-Refs';
```

Comment:
typo? `SEGMENT_FEATURE` vs `SEGMENT_REFS_FEATURE`?

Reply:
typo - will change it